tests/gold_tests/continuations/double.test.py

```diff
@@ -50,15 +48,18 @@
     'map http://127.0.0.1:{0} http://127.0.0.1:{1}'.format(ts.Variables.port, server.Variables.Port)
 )
 
-numberOfRequests = randint(1000, 1500)
+cmdstr = 'curl -k -vs http://127.0.0.1:{0}'.format(ts.Variables.port)
```

Comment:
We are requesting verbose output. Where is this output used? Or is it just for logging/debugging?

Reply:
Just for extra logging to give more info. This is incase the test can't `curl` ATS and that's what's actually causing the test to fail rather than the actual session, transaction count.

Reply:
OK sounds good.